data/human/news.txt

```diff
@@ -588,6 +595,8 @@ news "housekeeper"
 			"I just got fired. The dog of one of the kids died and the missus asked me to fly to Earth and buy a new identical one. When I returned she was screaming that she didn't need two dead dogs."
 			"There's one law for the rich and another law for the poor. One small mistake by me and I'm back to the horrible place I emigrated from."
 			"I don't know what part of 'no' those spoiled kids didn't understand, but I'm not cooking 5 course dinners every day when they eat almost nothing."
+			"I wanted to order one salt shaker, but things are apparently measured in tons when shipping stuff, so I got a ton of salt shakers. You want one? Or ten?"
+			"My employer wanted to donate some out-of-fashion dresses to starving people, but now she is angry because I said that three starving persons would fit in one of her dresses."
```

Comment:
```suggestion
			"My employer wanted to donate some out-of-fashion dresses to starving people, but now she's angry because I said that three starving people would fit in one of her dresses."
```